ctapipe/instrument/camera/geometry.py

```diff
@@ -800,7 +816,9 @@ def position_to_pix_index(self, x, y):
     @staticmethod
     def simtel_shape_to_type(pixel_shape):
         try:
-            return SIMTEL_PIXEL_SHAPES[pixel_shape]
+            shape, rotation = SIMTEL_PIXEL_SHAPES[pixel_shape]
+            # make sure we don't introduce a mutable global state
+            return shape, rotation.copy()
```

Comment:
Why do we need the `.copy()`?

Reply:
angles are mutable and this injected a global, mutable variable into classes that should not have any reliance on global state.